README.md

```diff
@@ -21,12 +21,12 @@
 
 ## Introduction
 
-Jujutsu is a powerful [version control system](https://en.wikipedia.org/wiki/Version_control)
-for software projects. You use it to get a copy of your code, track changes
-to the code, and finally publish those changes for others to see and use.
-It is designed from the ground up to be easy to use—whether you're new or
-experienced, working on brand new projects alone, or large scale software
-projects with large histories and teams.
+Jujutsu (呪術 _Jujutsu_) is a powerful [version control
+system](https://en.wikipedia.org/wiki/Version_control) for software projects.
+You use it to get a copy of your code, track changes to the code, and finally
+publish those changes for others to see and use. It is designed from the ground
+up to be easy to use—whether you're new or experienced, working on brand new
+projects alone, or large scale software projects with large histories and teams.
```

Comment:
I just went ahead and reflowed this paragraph, but the only actual real change is showing the kanji and romanized spelling.